static/js/util/util.js

```diff
@@ -50,6 +51,38 @@ export function makeCourseStatusDisplay(course, now = moment()) {
     courseOrRun = course.runs[0];
   }
 
+  let onEnrollBtnClick = (event) => {
+    event.preventDefault();
+    let url = [];
+    if (courseOrRun.course_id) {
+      url.push(
+        SETTINGS.edx_base_url,
+        'courses/',
+        courseOrRun.course_id,
+        '/info'
+      );
+      window.open(url.join(""), '_blank');
+    } else {
+      alert("Please enter valid course id");
```

Comment:
also I think we don't want to lean on `alert` here. in https://github.com/mitodl/micromasters/commit/0e8cc743c2354a1c79d631456176e6bcd25b70ea I added support for using the redux store to set text to show in a dialog, if we really want to alert the user I think that's a better option.